src/applications/lgy/coe/containers/IntroductionPage.jsx

```diff
@@ -1,102 +1,41 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 
 import { focusElement } from 'platform/utilities/ui';
 import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
-import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
 
-class IntroductionPage extends React.Component {
-  componentDidMount() {
+import { FormApplicationSteps } from '../components/FormApplicationSteps';
+
+function IntroductionPage(props) {
```

Comment:
This file is largely just conversion from class to functional component with some refactoring.